docs/bundler/css.md

```diff
@@ -5,6 +5,16 @@ Bun's bundler has built-in support for CSS with the following features:
 - CSS Modules
 - Tailwind (via a native bundler plugin)
 
+## CSS Chunking
+
+When multiple entry points import the same CSS files, use `--css-chunking` to reduce duplication by grouping shared CSS into separate chunks:
+
+```bash
+bun build ./home.ts ./about.ts --css-chunking --outdir ./dist
+```
+
+This creates separate CSS files for shared imports, reducing total CSS size in multi-page applications.
+
```

Comment:
this is nice as it fixes a few issues i believe, but i don't think its on Bun.build yet